### PR TITLE
fix format for $folder->getFile

### PR DESCRIPTION
**Pull request by franzholz**
I hope this will fix it.

### CODE REVIEW

File: Documentation/ApiOverview/Fal/UsingFal/ExamplesFileFolder.rst [hunk 1]
```diff
@@ -35,9 +35,9 @@ or by filename from its folder::
    $file = $folder->getStorage()->getFileInFolder("example.ext", $folder);
 
 .. versionadded:: 10.2
-   Starting with version 10.2  a file can be retrieved directly by its filename from the folder::
+   Starting with version 10.2 a file can be retrieved directly by its filename from the folder::
 
-      $file = $folder->getFile("filename.ext");
+   $file = $folder->getFile("filename.ext");
```
Comment (mbrodala) [inline Documentation/ApiOverview/Fal/UsingFal/ExamplesFileFolder.rst:40]:
```suggestion
   .. code-block:: php
   
      $file = $folder->getFile("filename.ext");
```

Hopefully this fixes the code block rendering.

Comment (franzholz) [reply Documentation/ApiOverview/Fal/UsingFal/ExamplesFileFolder.rst:41]: I have added ".. code-block:: php"

Comment (franzholz) [reply Documentation/ApiOverview/Fal/UsingFal/ExamplesFileFolder.rst:41]: I have used the same as you, but with 3 spaces indentation only.

And the "::" must be changed into ":".

Comment (mbrodala) [reply Documentation/ApiOverview/Fal/UsingFal/ExamplesFileFolder.rst:41]: Yes, you need to use 6 spaces instead:

1. 3 for the `..versionadded`
2. 3 for the `..code-block`

Comment (mbrodala) [reply Documentation/ApiOverview/Fal/UsingFal/ExamplesFileFolder.rst]: Not necessary if a text block is 1st level, then `::` can be used just like before.

It's just in this case that this shortcut apparently does not work within `.. versionadded`.

Comment (franzholz) [reply Documentation/ApiOverview/Fal/UsingFal/ExamplesFileFolder.rst]: Yes, it is not necessary, but I consider it better to have the same Sphinx controls on one page.

Comment (mbrodala) [reply Documentation/ApiOverview/Fal/UsingFal/ExamplesFileFolder.rst]: And I prefer minimal changes, which means _not_ changing everything on this page to have the same style.

So how to proceed now?

File: Documentation/ApiOverview/Fal/UsingFal/ExamplesFileFolder.rst [hunk 2]
```diff
@@ -35,8 +35,9 @@ or by filename from its folder::
    $file = $folder->getStorage()->getFileInFolder("example.ext", $folder);
 
 .. versionadded:: 10.2
-   Starting with version 10.2  a file can be retrieved directly by its filename from the folder::
-
+   Starting with version 10.2 a file can be retrieved directly by its filename from the folder:
+   .. code-block:: php
```
Comment (mbrodala) [inline Documentation/ApiOverview/Fal/UsingFal/ExamplesFileFolder.rst:39]:
```suggestion

   .. code-block:: php
```

**Pull request merged**